projects/core/src/routing/services/activated-routes.service.spec.ts

```diff
@@ -65,7 +64,7 @@ describe('ActivatedRoutesService', () => {
     });
 
     it('should emit array of activated routes', async () => {
-      const mockRouterState: RouterState = <RouterState>{
+      const mockRouterState: any = {
```

Comment:
What's wrong with RouterState?

Reply:
RouterStateSnapshot  has some changes in Angular 14